### ISSUE TITLE
API: Rollback secret test failing

### ISSUE THREAD

**Issue by rajdip-b** [keyshade-xyz/keyshade#295]
## Description

https://github.com/keyshade-xyz/keyshade/blob/68f51a5121bb28496b06984438be0bbacc6b60a6/apps/api/src/secret/secret.e2e.spec.ts#L544

Inspect why this test is failing.

**Comment by adityaRajGit** [comment]
Can I try?

**Comment by rajdip-b** [comment]
Sure!

**Comment by rajdip-b** [comment]
@adityaRajGit unassigning due to inactivity

**Comment by Fahad-Dezloper** [comment]
Is this issue still open?

**Comment by rajdip-b** [comment]
Yes!

**Comment by rajdip-b** [comment]
Yeah! i noticed this just yesterday. I am working on a PR where i have already addressed this issue. I didn't remember we had an open issue for that. But thanks a lot for pointing out!
